app/components/container/containerController.js

```diff
@@ -48,38 +48,21 @@ function ($q, $scope, $state, $transition$, $filter, Container, ContainerCommit,
     });
   };
 
-  $scope.start = function () {
+  var doAction = function (Action, Id) {
     $('#loadingViewSpinner').show();
-    Container.start({id: $scope.container.Id}, {}, function (d) {
-      update();
-      Notifications.success('Container started', $transition$.params().id);
-    }, function (e) {
-      update();
-      Notifications.error('Failure', e, 'Unable to start container');
-    });
-  };
-
-  $scope.stop = function () {
-    $('#loadingViewSpinner').show();
-    Container.stop({id: $transition$.params().id}, function (d) {
-      update();
-      Notifications.success('Container stopped', $transition$.params().id);
+    Container[Action]({id: Id}, {}, function (d) {
+      update(); Notifications.success('Container '+Action+'ed', $transition$.params().id);
```

Comment:
I believe we should add a separate function to convert word from Present to Past tense because in case of "stop" action it should be a "stopped" instead of "stoped"

Reply:
Done. I added a exception for "stop" only. I think that a more complete implementation (such as https://gist.github.com/letsgetrandy/1e05a68ea74ba6736eb5) would be overkill.

Reply:
Let's move new function to the separate service since it isn't related to the controller itself and we could reuse it in other places as well.

Indeed that solution would be an overkill for now but we could start with a mapping for exceptions(for now it will just contain "stop" mapping) and default rule.